README.rdoc

```diff
@@ -2,32 +2,38 @@
 
 Rails gem pagination helper that works off of scopes (named) to facilitate data retrieval and display.
 
-== Install 
+== Install
 
   Use the bundler gem (sudo gem install bundler)
   Add the following to your Gemfile
 
     gem "pagem", :git => "git://github.com/mdsol/pagem.git"
-  
+
+== Testing
+
+  rvm use  ruby-1.9.3-p125
+  bundle install
+  bundle exec rspec spec/pagem_spec.rb
+
```

Comment:
Thanks for the docs! Is the Gemfile.lock change intentional, and could we have a new test for the changes here?